run_dosbox

```diff
@@ -85,7 +85,8 @@ def run_dosbox(args):
 def run_dosbox_with_conf(args):
     name = confgen.uniq_conf_name(STEAM_APP_ID, args)
     if settings.get_confgen_force() or not os.path.isfile(name):
-        print_err(f'run_dosbox: generating {name} based on {args}')
+        print_err('run_dosbox: generating {0} based on {1}'.format(
+            name, args))
```

Comment:
I am a bit annoyed that neither linter did not caught this :/

Fix line 89 so that first parameter is aligned ('name' should be in the same column as 'run_dosbox…').

I need to find a way to enforce this through automatic check.

Reply:
Interesting! I've never known anyone to consider this a problem before (not that I know a ton of Python developers mind you), but it's never even crossed my mind. :sweat_smile:

I just skimmed over PEP8 again to be sure, but it doesn't make it clear IMO that there's anything wrong with this AFAICT, since `name` and `args` aren't actually arguments to `print_err()`, but I think by indenting them in the way you suggest implies that they are to someone skimming the code.

In fact, if I hit tab on that second line to switch between all the different indentation types in Emacs python-mode, indenting to align with the start of the string is never even given as an option.

So I decided to look to what I imagine is the authoritative source (pardon the pun) on the matter - the cpython library, and here are a few examples I found there that are written exactly the same way.

https://github.com/python/cpython/blob/v3.5.7/Lib/enum.py#L106
https://github.com/python/cpython/blob/v3.5.7/Lib/functools.py#L685
https://github.com/python/cpython/blob/v3.5.7/Lib/traceback.py#L392

It's basically treated no differently from lines like this:
https://github.com/python/cpython/blob/v3.5.7/Lib/traceback.py#L276
https://github.com/python/cpython/blob/v3.5.7/Lib/collections/__init__.py#L900

I'm happy to change it if you're not convinced (I don't personally mind - I'm just grateful you're considering my patch), but it feels strange to me to do that.

Reply:
This is another option that I believe is valid and fits, although I find it to look slightly odd:
```
        print_err(
            'run_dosbox: generating {0} based on {1}'.format(name, args))
```

Reply:
The joy of style-related conversations ;) 

I am more used to kernel C-style (not python, but it's just argument passing) and Google style (python and otherwise) - that's probably why I noticed this immediately. Emacs uses GNU style IIRC. Don't worry about it - worst case I will fix it automatically with one of future commits.

pylint and pycodestyle do not even have an option to detect this quirk - but it turns out https://github.com/google/yapf does (aligns parameters by default - both for 'pep-8' and 'google' stylesheets). In future I will integrate yapf formatter to limit style-related discussions to minimum  :)

Reply:
That's fair enough. I wasn't aware of yapf. I've always just used pylint3 and never felt the need to look into alternatives, but pylint3 does complain when I make that change:
```
C: 89, 0: Wrong hanging indentation (remove 6 spaces).
                  name, args))
            |     ^ (bad-continuation)
```
Not sure if you want me to add a `disable=` comment to silence it, or if pylint3 warnings are going to be quite common when using yapf anyway? I see you do already have the `pylint: disable=missing-docstring` line there so am not sure. In any case, I'm happy to take your suggestion and let you change it in a future commit the way you see fit. :+1:

Reply:
Ah, indeed it does complain. That's because I missed that these are arguments to `.format` and not to `print`. That's fine.

As for pylint - there's also `.pylint` file where I configure whatever needs to be configured, add overrides for dictionary words, etc. But generally I prefer to not add spurious pylint overrides.